src/validations/vulnerable-dependencies.js

```diff
@@ -19,14 +20,36 @@ module.exports = {
 
     run () {
         return new Promise((resolve, reject) => {
-            const pkg = pathJoin(process.cwd(), 'package.json');
+            const projectDir = pathJoin(process.cwd());
+            const defaultArgs = nsp.sanitizeParameters({});
+            const args = {
+                baseUrl:     defaultArgs.baseUrl,
+                proxy:       defaultArgs.proxy,
+                reporter:    'summary',
+                'warn-only': false,
+                path:        projectDir,
+                pkg:         readPkg.sync(projectDir, { normalize: false }),
+                offline:     false,
+                exceptions:  []
+            };
 
-            nsp.check({ package: pkg }, (err, data) => {
-                if (err || data.length > 0)
-                    reject(nsp.formatters.summary(err, data));
-                else
-                    resolve();
-            });
+            nsp.check(args)
+                .then(result => {
+                    if (result && result.data && result.data.length === 0) {
+                        resolve();
+                        return;
+                    }
+                    if (result && result.data && result.data.length > 0) {
```

Comment:
Duplicate `result && result.data` check: can be either moved to a boolean variable or common `if` statement.

Reply:
Hi @inikulin, Thnks for your feedback. I implement your recommandations asap